src/main/java/duke/Event.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents an event task with a description, start time, and end time.
+ * Extends the Task class and includes methods for creating and displaying event tasks.
+ */
+public class Event extends Task {
+
+    protected LocalDateTime start;
+    protected LocalDateTime end;
+
+    /**
+     * Creates a new event task with the specified description, start time, and end time.
+     *
+     * @param description The description of the event.
+     * @param start       The start time of the event in "yyyy-MM-dd HHmm" format.
+     * @param end         The end time of the event in "yyyy-MM-dd HHmm" format.
+     * @throws DukeException If the description, start time, or end time is empty.
+     */
+    public Event(String description, String start, String end) throws DukeException {
+        super(description, TaskType.EVENT);
```

Comment:
should description be .trim() here as well?
```suggestion
        super(description.trim(), TaskType.EVENT);
```